main.css

```diff
@@ -187,3 +187,10 @@
 .dashboard .news .follow .Box.p-3.mt-2 {
   display: none;
 }
+
+/**
+ * Remove expired academic discount coupon alert box.
+ */
+.flash {
```

Comment:
Does this have a parent container like `.dashboard`? If yes, we need to do something like `.dashboard .flash` in order to only hide this from the dashboard. `.flash` might affect the whole of GitHub.